<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRelReferenceConfigDlgBase</class>
 <widget class="QWidget" name="QgsRelReferenceConfigDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Display expression</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="mTxtDisplayExpression"/>
     </item>
     <item>
      <widget class="QToolButton" name="mPbnPreviewExpression">
       <property name="text">
        <string>Expression Builder</string>
       </property>
       <property name="icon">
        <iconset resource="../../../images/images.qrc">
         <normaloff>:/images/themes/default/mIconExpressionEditorOpen.svg</normaloff>:/images/themes/default/mIconExpressionEditorOpen.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="mComboRelation"/>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="mCbxShowForm">
     <property name="text">
      <string>Show embedded form</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Relation</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="mCbxAllowNull">
     <property name="text">
      <string>Allow NULL value</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
